tests/pipelines/test_pipelines_token_classification.py

```diff
@@ -580,6 +594,41 @@ def test_gather_pre_entities(self):
             ],
         )
 
+    @require_torch
+    def test_word_heuristic_leading_space(self):
+        model_name = "microsoft/deberta-base"
```

Comment:
I think if we can use `hf-internal-testing/tiny-random-deberta-v2`  it'd be a much smaller model which are more targeted fast tests.

It should use the same tokenizer and expose the same things.

Reply:
Yes, good call, will update